remix-solidity/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "include": ["src", "index.ts"],
+    "compilerOptions": {
+      "target": "es6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+      "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+      "lib": ["dom", "es2018"],                 /* Specify library files to be included in the compilation. */
+      "declaration": true,                      /* Generates corresponding '.d.ts' file. */
+      "sourceMap": true,                        /* Generates corresponding '.map' file. */
+      "outDir": "./dist",                       /* Redirect output structure to the directory. */
+      /* Strict Type-Checking Options */
+      "strict": true,                           /* Enable all strict type-checking options. */
+      "noImplicitAny": false,                  /* Raise error on expressions and declarations with an implied 'any' type. */
+      /* Module Resolution Options */
+      "baseUrl": "./src",                       /* Base directory to resolve non-absolute module names. */
+      "paths": { "remix-solidity": ["./"] },       /* A series of entries which re-map imports to lookup locations relative to the 'baseUrl'. */
+      "typeRoots": [
+        "./@types",
+        "./node_modules/@types"
+      ],
+      "esModuleInterop": true,                   /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
+      /* Experimental Options */
+      "experimentalDecorators": true,           /* Enables experimental support for ES7 decorators. */
```

Comment:
Are you using decorators ? If not set it to false.

Reply:
updated